### DISCUSSION TITLE
How to build only one project?

### DISCUSSION THREAD

**Discussion by SouEuMarlon** [maizzle/framework#947] [Q&A]
Hello guys, how's you going?

I have menny projects in project folder and when I'm workin in a project and I save it, all others projects be building too.
What can I do to build just the project I'm currently working on?

**Comment by cossssmin** [comment]
You could define one config.js per project, so each config is used to build a certain project.

For example if you have `src/templates/client1`, you'd create `config.client1.js` and set `build.templates.source` to that folder.

Running `maizzle build client1` would only build those templates. And you can also do `maizzle serve client1` to develop locally only those templates.

Docs: https://maizzle.com/docs/environments#custom-environments

**Reply by SouEuMarlon** [reply]
I didn't even think I had that possibility. Thanks a lot for the help.
